ts/features/itwallet/common/components/ItwCredentialNotFound.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from "react";
+import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import I18n from "../../../../i18n";
+import { ItwCredentialIssuanceMachineContext } from "../../machine/provider";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { useItwDisableGestureNavigation } from "../hooks/useItwDisableGestureNavigation";
+import { useAvoidHardwareBackButton } from "../../../../utils/useAvoidHardwareBackButton";
+
+const ItwCredentialNotFound = ({
+  credentialType
+}: {
+  credentialType: string;
+}) => {
+  const machineRef = ItwCredentialIssuanceMachineContext.useActorRef();
+  const navigation = useIONavigation();
+
+  // Disable the back gesture navigation and the hardware back button
+  useItwDisableGestureNavigation();
+  useAvoidHardwareBackButton();
+
+  const navigateToCredential = () => {
+    machineRef.send({
+      type: "select-credential",
+      credentialType,
+      skipNavigation: true
+    });
+  };
+
+  const handleClose = () => {
+    navigation.pop();
+  };
+
+  // Since this component could be used on a screen where the header is visible, we hide it.
+  useEffect(() => {
+    navigation.setOptions({
+      headerShown: false
+    });
+  }, [navigation]);
+
+  return (
+    <OperationResultScreenContent
+      pictogram="cie"
+      title={I18n.t("features.itWallet.issuance.credentialNotFound.title")}
+      subtitle={I18n.t(
+        "features.itWallet.issuance.credentialNotFound.subtitle"
+      )}
+      isHeaderVisible={false}
+      action={{
+        label: I18n.t("global.buttons.continue"),
+        accessibilityLabel: I18n.t("global.buttons.continue"),
+        onPress: navigateToCredential
+      }}
```

Comment:
We should add a loading state to this button when we send the issuance request (machine is loading). What do you think? Currently we do not have any feedback

Reply:
Nevermind: I forgot that we can pass `skipNavigation: false` to the `select-credential` event to see a loading screen 😅

Reply:
Fixed, removed the prop to skip the loading!